src/index.js

```diff
@@ -3,16 +3,23 @@ import {
   createLeaderboardListElement,
   cleanForms,
   appendElementToLeaderboard,
-} from './func-module';
+} from './func-module.js';
+
+import { getScoresFromAPI, setScoresToAPI } from './api-module.js';
 
 const addScoreBtn = document.getElementById('add-score-button');
+const refreshBtn = document.getElementById('refresh-button');
 
 addScoreBtn.addEventListener('click', () => {
   const newName = document.getElementById('name-form').value;
   const newScore = document.getElementById('score-form').value;
```

Comment:
- [OPTIONAL] It will be nice to properly validate the `name` and `score` fields, so that the user can not submit blank entries, or wrong data types.